### DISCUSSION TITLE
set_parameter?

### DISCUSSION THREAD

**Discussion by FGasper** [pmqs/Compress-Stream-Zstd#1] [Ideas]
@pmqs: Compress::Zstd’s maintainer appears to be inactive.

Would you have any interest in merging https://github.com/spiritloose/Compress-Zstd/pull/25 into your project?

My erstwhile interest was in creating a Zstd compression extension for WebSocket. I wanted to use Perl to create a PoC but never got back to it because I wanted set_parameter to merge first.

Thanks!

**Comment by pmqs** [comment]
Hey @FGasper 

Yes, I can try merging you PR. It ticks all the boxes as it has both updated documentation and tests! Not sure how to get the change into my fork though. Any chance you could create a new one?

My intention when I forked this repo, was to eventually push my changes back upstream, and retire my fork,  but it doesn't look like that will happens any time soon.

**Comment by pmqs** [comment]
I've figured it out. Merge in process

**Comment by pmqs** [comment]
Uploaded code with you change to Git & CPAN. Ping me if yu have any issues

**Comment by FGasper** [comment]
Awesome, @pmqs—thank you! :)
